frontend/src/concepts/pipelines/context/PipelinesContext.tsx

```diff
@@ -142,11 +158,21 @@ export const usePipelinesAPI = (): UsePipelinesAPI => {
     timedOut: serverTimedOut,
   };
 
+  const getJobInformation: GetJobInformation = () => {
+    const jobStatus: JobStatus = {
+      loading: true,
+      data: null,
+    };
+
+    return jobStatus;
+  };
```

Comment:
Delete this -- thread the value from:
```
  const {
    hasCR,
    crInitializing,
    serverTimedOut,
    apiState,
    namespace,
    project,
    refreshAPIState: refreshAllAPI,
  } = React.useContext(PipelinesContext);
```

So once you delete it -- the value you import will take its place:
```
const {
  ...
  getJobInformation,
} = React.useContext(PipelinesContext);
```
And this should make it work.